lib/purl_fetcher/client/reader.rb

```diff
@@ -18,11 +19,17 @@ def collection_members(druid)
     end
   end
 
+  # @return [Array<Hash<String,String>>] a list of hashes where the key is a digest and the value is a filepath/filename
+  def files_by_digest(druid)
+    get("/purls/druid:#{druid.delete_prefix('druid:')}", {})
+      .fetch("files_by_md5", [])
+  end
+
   private
 
   ##
   # @return [Hash] a parsed JSON hash
-  def fetch(path, params)
+  def get(path, params)
```

Comment:
To make this a little clearer.

Reply:
I'm not convinced this is clearer. Here I'm confused whether this refers to the verb "go get data" or specifically to the HTTP "GET" method.

Reply:
I'm trying to avoid this above:

```
fetch(url, {})
  .fetch("files_by_md5", [])
```

How about I rename this method `http_get`?

Reply:
Ugh....yeah, two versions of "fetch" isn't better.